cpp_chapters/what_is_a_program.md

```diff
@@ -28,17 +35,18 @@ int main() {
 }
 ```
```

Comment:
It is worthwhile to add the new vesion of this:
The appropriate markdown should look like this

`````````Markdown
``````{tab} Pre C++20
```c++
#include <iostream>
int main() {
    std::cout << "Hello, World!" std::endl;
    return 0;
}
```
``````
``````{tab} Post C++20
```c++
import <iostream>;
auto main() -> int {
    std::cout << "Hello, World!" std::endl;
    return 0;
}
```
Assuming this file is names `main.cpp`, it can be compiled using the command
```bash
g++ -std=c++20 -fmodels-ts -o main main.cpp -x c++-system-header iostream
```
``````
``````{tab} Post C++23
```c++
import <print>;
auto main() -> int {
    std::print("Hello, World!");
    return 0;
}
```
``````
`````````